src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitor.scala

```diff
@@ -87,11 +87,18 @@ class SubmissionMonitor(workspaceName: WorkspaceName,
   }
 
   private def startWorkflowMonitorActors(): Unit = {
-    submission.workflows.filterNot(workflow => workflow.status.isDone).foreach(startWorkflowMonitorActor(_))
+    datasource.inTransaction(readLocks=Set(workspaceName)) { txn =>
```

Comment:
to be a little less terse, something at startup still needs to load all the submissions that are not in "Done" state and send the SubmissionSupervisor a SubmissionStarted message.